version.sbt

```diff
@@ -1,12 +1 @@
-import java.io.FileInputStream
-import java.util.Properties
-
-version in ThisBuild := {
-  val s = new FileInputStream(file("version.properties"))
-  try {
-    val p = new Properties()
-    p.load(s)
-    p.getProperty("version")
-  } finally s.close()
-
-}
+version in ThisBuild := "0.26.0-SNAPSHOT"
```

Comment:
See https://github.com/sbt/sbt-typesafe-conductr/pull/57#discussion_r27727566 
During review it got a "great", so we merged.